### PR TITLE
feat(snap): Allow RTSP server config using snap options

### PR DESCRIPTION
**Pull request by farshidtz**
This change is to allow RTSP server configuration using snap options.

At the same time, the RTSP server is being restricted to run only a RTSP/TCP listener that binds to the loopback interface.

The following listeners are being disabled by default:
- RTP
- RTCP
- multicastRTP
- multicastRTCP
- RTMP
- HLS

For details, refer to https://github.com/aler9/rtsp-simple-server/blob/main/rtsp-simple-server.yml

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports
Are there any new imports or modules? If so, what are they used for and why?

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

```
$ snap start edgex-device-usb-camera.rtsp-simple-server 
$ snap logs edgex-device-usb-camera.rtsp-simple-server
...
[RTSP] listener opened on 127.0.0.1:8554 (TCP)

# Add UDP to protocols
$ snap set edgex-device-usb-camera apps.rtsp-simple-server.config.rtsp-protocols="tcp,udp"
$ snap start edgex-device-usb-camera.rtsp-simple-server 

# Verify that the UDP listener are started
$ snap logs edgex-device-usb-camera.rtsp-simple-server
...
[RTSP] listener opened on 127.0.0.1:8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
```

## Other information

### CODE REVIEW

File: snap/local/helper-go/go.mod [hunk 1]
```diff
@@ -1,11 +1,7 @@
...
+module github.com/edgexfoundry/device-usb-camera/snap/local/helper-go
 
 require github.com/canonical/edgex-snap-hooks/v2 v2.3.0
 
-require (
-	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/stretchr/testify v1.7.1 // indirect
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-)
+replace github.com/canonical/edgex-snap-hooks/v2 => github.com/farshidtz/edgex-snap-hooks/v2 v2.0.6-0.20220714112554-027cc49df5a3
```
Comment (farshidtz) [inline snap/local/helper-go/go.mod]: Pending merge and release of https://github.com/canonical/edgex-snap-hooks/pull/67

**Review by MonicaisHer — APPROVED** [review]
@farshidtz Thank you, looks good.
I have tested this snap using snap options to configure RTSP server, and the RTSP server is being restricted to run only an RTSP/TCP listener that binds to the loopback interface. Everything works as expected.

**Comment by farshidtz** [comment]
recheck

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Pull request merged**